docs/en/data_types/domains/domains.md

```diff
@@ -0,0 +1,24 @@
+# Domains
```

Comment:
* New files should be added into `toc_en.yml`
* Not-yet-translated languages should have similar files as relative symlinks to English documents and also added to `toc_*.yml`
* domains/domains.md will have weird URL with double "domains", so it'd be better to rename to overview.md or something like that

Reply:
Fixed